core/components/homepage-slideshow/index.tsx

```diff
@@ -0,0 +1,50 @@
+import SlideshowBg01 from '~/components/homepage-slideshow/slideshow-bg-01.jpeg';
+import SlideshowBg02 from '~/components/homepage-slideshow/slideshow-bg-02.jpeg';
+import SlideshowBg03 from '~/components/homepage-slideshow/slideshow-bg-03.jpeg';
```

Comment:
Since these are scoped to this directory, maybe we name them `slide-bg-01.jpeg`, etc.?